napari_arboretum/tree.py

```diff
@@ -6,10 +6,11 @@
 
 # colormaps
 turbo = AVAILABLE_COLORMAPS["turbo"]
-WHITE = np.array([255, 255, 255, 255], dtype=np.uint8)
-RED = np.array([255, 0, 0, 255], dtype=np.uint8)
+WHITE = np.array([1, 1, 1, 1], dtype=float)
+RED = np.array([1, 0, 0, 1], dtype=float)
```

Comment:
Why are you changing the type?

Reply:
Because these are now in the range [0, 1] instead of [0, 255].